src/Particle/Par_Init_Attribute_Mesh.cpp

```diff
@@ -73,53 +73,59 @@ void Par_Init_Attribute_Mesh()
    char FileName[] = "Input__Par_Mesh";
 
 // retrieve the number of mesh quantities specified in Input__Par_Mesh
-   const int Mesh_NAttr = Aux_CountRow( FileName );
+   const int Mesh_NAttr = ( Aux_CheckFileExist(FileName) ) ? Aux_CountRow( FileName ) : 0;
```

Comment:
Agreed. 

Since it is already done in `Aux_CountRow()`, I reset this line to the previous version. Also, I add the `return` statement if `Mesh_NAttr = 0` such that we do not need check it again in the following code.